lib/plugins/aws/provider/awsProvider.js

```diff
@@ -524,7 +524,7 @@ class AwsProvider {
   }
 
   getLogRetentionInDays() {
-    if (!_.has(this.serverless.service.provider, 'logRetentionInDays')) {
+    if (!('logRetentionInDays' in this.serverless.service.provider)) {
```

Comment:
Let's put it as `if (!this.serverless.service.provider.logRetentionInDays)`

Reply:
Hmm, this will need one more condition eventually.
In `mergeIamTemplates.test.js@633`, `logRetentionInDays` is set to 0, so `if (!0)` will be `true` and the test will fail. I should check if `logRetentionInDays` is bigger than 0, maybe add one more test case with a negative value.

Reply:
Let's remove problematic test. We're shifting away from such internal validation, as all will be upfront validated via config schema (handled at https://github.com/serverless/serverless/issues/6562 )